lib/ace/snippets.js

```diff
@@ -914,8 +915,6 @@ var TabstopManager = function(editor) {
         var ranges = this.ranges;
         tabstops.forEach(function(ts, index) {
             var dest = this.$openTabstops[index] || ts;
-            ts.rangeList = new RangeList();
-            ts.rangeList.$bias = 0;
             
             for (var i = 0; i < ts.length; i++) {
                 var p = ts[i];
```

Comment:
Not related to this CR, but if we want to keep using Ace in the long term, I feel the code has to be a lot more readable. 

To make this more readable, I would expect an explanation of the TabStop data structure somewhere, explaining that one tabstop can have multiple ranges, one for each instance of a variable. I think a practical way of adding this documentation is by creating a TabStop class and putting it there.